components/scream/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ add_subdirectory(${SCREAM_BASE_DIR}/../../externals/YAKL ${CMAKE_BINARY_DIR}/ext
 target_compile_options(yakl PUBLIC $<$<COMPILE_LANGUAGE:CXX>:${YAKL_CXX_FLAGS}>)
 EkatDisableAllWarning(yakl)
 
+# For debug builds, set -DYAKL_DEBUG                                                                                       
+string(TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE_ci)                                                                
```

Comment:
FYI, we are already computing this in scream's main CMakeLists.txt, storing the result in `SCRAM_DEBUG`.

Reply:
Is it better to use SCREAM_DEBUG here then? I thought it might be more appropriate to use the CMAKE intrinsic CMAKE_BUILD_TYPE directly, rather than relying on the SCREAM-specific flag to make this more portable.